src/components/businesses/BusinessesPage.js

```diff
@@ -13,6 +14,17 @@ class BusinessesPage extends Component {
     );
   }
 
+  _handleOnChangeFilterOptions = (type, name) => {
+    const filters = this.props.businesses.appliedFilters.category;
+    if (isEmpty(filters)) {
```

Comment:
This can be refactored.

The `isEmpty` check is useless because it will still call `this.props.handleOnChangeFilterOptions(type, name)`. Just filter the `filters` else call `this.props.handleOnChangeFilterOptions(type, name)`.